src/platform/site-wide/cta-widget/tests/index.unit.spec.js

```diff
@@ -568,6 +565,63 @@ describe('<CallToActionWidget>', () => {
         expect(tree.find('NeedsVAPatient').exists()).to.be.true;
         tree.unmount();
       });
+
+      describe('ssoe', () => {
+        const ssoeProps = { ...{ useSSOe: true }, ...defaultProps };
```

Comment:
nitpick, this can just be
```js
const ssoeProps = {...defaultProps, useSSOe: true}
```
I can fix this

Reply:
oh cool, thanks